src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractAccessControlNameCheck.java

```diff
@@ -95,6 +96,22 @@ protected boolean shouldCheckInScope(DetailAST modifiers) {
                 || applyToPrivate && isPrivate;
     }
 
+    /**
+     * Checks if given modifiers has public access.
+     * There are 2 cases - it is either has explicit modifier, or it is
+     * in annotation or interface.
+     *
+     * @param modifiers - modifiers to check
+     * @return true if public
+     */
+    private static boolean isPublic(DetailAST modifiers) {
+        return modifiers.findFirstToken(TokenTypes.LITERAL_PUBLIC) != null
+                || ScopeUtil.isInAnnotationBlock(modifiers)
+                || ScopeUtil.isInInterfaceBlock(modifiers)
+                // interface methods can be private
+                && modifiers.findFirstToken(TokenTypes.LITERAL_PRIVATE) == null;
```

Comment:
+2 in indentation, as it is not on same level of execution as `||`

Reply:
done